sdks/python/test-suites/dataflow/build.gradle

```diff
@@ -0,0 +1,50 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: org.apache.beam.gradle.BeamModulePlugin
+applyPythonNature()
+
+dependencies {
+  distTarBall project(path: ":sdks:python", configuration: "distTarBall")
+}
+
+task preCommitIT {
+  getListProperty('dataflow_precommit_it_task_py_versions').each {
```

Comment:
I think for a reader `getListProperty` name might be confusing  and they would have to look up the implementation. We could consider the name `getVersionsAsList` or use `getProperty('dataflow_precommit_it_task_py_versions').split(',').each` directly.

Reply:
Quite so. I use `getVersionsAsList` instead of `getListProperty`.